### DISCUSSION TITLE
Package name

### DISCUSSION THREAD

**Discussion by benmerckx** [electric-sql/pglite#304] [Ideas]
The 'pglite' package name on npm is still free. It's easier to remember than '@electric-sql/pglite' :)

**Comment by samwillis** [comment]
Hey @benmerckx 

Thanks for the suggestion. I tried to use the "pglite" package name when we launched but npm rejected it as too similar to the existing "pg-lite" package.

Now that we have some traction, and as that package is clearly abandoned we may be able to request it. I will look into it!
